examples/tutorials/ItWaspsOnMyMachine/ext/queries.js

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
+
+export const getExcuse = async () => {
+     return axios
+          .get('https://api.devexcus.es/')
+          .then(res => {
+               return res.data;
+          })
+          .catch(error => {
+               console.error(error);
+          });
+}
+
+export const getAllExcuses = async (context) => {
+     return context.entities.Excuse.findMany()
```

Comment:
I get `xhr.js:187 POST http://localhost:3001/operations/get-all-excuses 500 (Internal Server Error)` errors in the browser console when calling this route. I think since you are using `context` as the first arg it is null, but it should be the second.

Reply:
Fix: `export const getAllExcuses = async (_args, context) => {`

Reply:
> * do we want this under `examples/tutorials` or just `examples`? In other words, will we actually build a tutorial around this, or will it just be another regular example? If the latter, you may want to move it up one level.

Yes, I've planned to make a tutorial on this app and add it to the documentation

> * do you want to add any Wasp branding anywhere? we have a Wasp header used in other examples.

Honestly, in this particular case, I'd like to avoid adding any additional line of code, that is not necessary. If you don't mind -- I'd opt out Wasp logo for now. 

> * do you want to deploy and host this anywhere?

Well, not yet :)